app/components/CrashDiagramCard.tsx

```diff
@@ -2,63 +2,164 @@ import { useState } from "react";
 import Alert from "react-bootstrap/Alert";
 import Card from "react-bootstrap/Card";
 import Image from "react-bootstrap/Image";
+import Button from "react-bootstrap/Button";
+import ButtonGroup from "react-bootstrap/ButtonGroup";
+import { BsZoomIn, BsZoomOut } from "react-icons/bs";
+import { SlActionUndo } from "react-icons/sl";
+import {
+  TransformWrapper,
+  TransformComponent,
+  useControls,
+} from "react-zoom-pan-pinch";
 import { Crash } from "@/types/crashes";
 
-// import axios from "axios";
-// import { TransformWrapper, TransformComponent } from "react-zoom-pan-pinch";
-
 const CR3_DIAGRAM_BASE_URL = process.env.NEXT_PUBLIC_CR3_DIAGRAM_BASE_URL!;
 
+const ZoomResetControls = ({
+  setRotation,
+}: {
+  setRotation: (value: number) => void;
+}) => {
+  const { zoomIn, zoomOut, resetTransform } = useControls();
+
+  const handleReset = () => {
+    resetTransform();
+    setRotation(0);
+  };
+
+  // pixel pushing
+  const iconStyle = {
+    position: "relative" as const,
+    top: "-1px",
+    fontSize: "1.3em",
+  };
+
+  return (
+    <div
+      className="tools"
+      style={{
+        display: "flex",
+        justifyContent: "space-between",
+        width: "100%",
+        marginBottom: "5px",
+      }}
+    >
+      <div>
```

Comment:
is this div necessary?

Reply:
It's what I use to slide the zoom buttons over to the left while leaving the undo button on the right.